simexpal/launch/common.py

```diff
@@ -139,6 +139,7 @@ def get_paths(self):
 		paths = []
 		for build_yml in self.yml['builds']:
 			paths.append(os.path.join(build_yml['prefix'], 'bin'))
+			paths.append(build_yml['prefix'])
```

Comment:
Why this change? That seems strange to me.

Reply:
There were cases where users had their executables directly in the prefix-dir.

Reply:
But isn't the fix to move them into `bin/` then? Or, we could add a `extra_paths` argument to allow the the user to configure this behavior. In any case, adding the prefix directory unconditionally does not sound like a good resolution to me.

Reply:
Ok, that sounds reasonable. I will remove this line and create an issue regarding the `extra_paths` argument so that I do not forget about it and implement it in the future.